packages/binding-coap/package.json

```
@@ -43,6 +43,7 @@
         "node-coap-client": "1.0.8",
         "rxjs": "5.5.11",
         "slugify": "^1.4.5",
+        "wot-thing-description-types": "^1.1.0-23-March-2023",
```

```suggestion
        "wot-thing-description-types": "1.1.0-23-March-2023",
```

I other package.json we use a fix version since in the past we sometimes run into issues.